lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -6,7 +6,7 @@
                              [attr.data-automation-id]="'adf-metadata-group-properties'">
             <mat-expansion-panel-header>
                 <mat-panel-title>
-                    <h1 class="adf-metadata-properties-title">{{ 'CORE.METADATA.BASIC.HEADER' | translate }}</h1>
+                    <h3 class="adf-metadata-properties-title">{{ 'CORE.METADATA.BASIC.HEADER' | translate }}</h3>
```

Comment:
This will create another accessibility issue, all h tags needs to be in order so if h3 can't be used if h1 and h2 are not present in the DOM tree

Reply:
<h1> tag changed to <h2> as per the discussion as <h1> is already present at the top level

Reply:
h1 tag changed to h2 as per the discussion as h1 is already present at the top level.